pkg/suggestion/suggest.go

```diff
@@ -74,7 +74,11 @@ func SubcommandsRequiredWithSuggestions(cmd *cobra.Command, args []string) error
 		return fmt.Errorf(requireMsg, typedName, cmd.CommandPath())
 	}
 
-	return cmd.Help()
+	if typedName == "" {
+		return cmd.Help()
+	}
+	fmt.Fprintf(cmd.ErrOrStderr(), "%s %s doesn't exists\n", cmd.Name(), typedName)
```

Comment:
@vdemeester @vinamra28 

So with this approach we get the following:

```
./tkn ctl
tkn ctl doesn't exists
CLI for tekton pipelines

Usage:
tkn [flags]
  tkn [command]

Available Commands:
  clustertask           Manage ClusterTasks
  clustertriggerbinding Manage clustertriggerbindings
  condition             Manage conditions
  eventlistener         Manage eventlisteners
  pipeline              Manage pipelines
  pipelinerun           Manage PipelineRuns
  resource              Manage pipeline resources
  task                  Manage Tasks
  taskrun               Manage TaskRuns
  triggerbinding        Manage triggerbindings
  triggertemplate       Manage triggertemplates

Other Commands:
  completion            Prints shell completion scripts
  version               Prints version information

Flags:
  -h, --help   help for tkn

Use "tkn [command] --help" for more information about a command.
```

1. This is difficult to find when located at the top because of the displaying of the help message.
2. It is difficult to find the error message since there is no `Error` prefix. 

I think an ideal solution to this would be as follows:

```
$ tkn ctl

Error: command "ctl" does not exist. Run tkn help for list of available commands.
```

Reply:
@danielhelfand while still showing the help right ?

Reply:
Without. I think something like this would make it easier to find the error:

```go
return fmt.Errorf("command %s %s doesn't exist. Run tkn help for available commands", cmd.Name(), typedName)
```

Would result in:
```
$ tkn ctl
Error: command tkn ctl doesn't exist. Run tkn help for available commands
```

Reply:
Alright, make sense too :upside_down_face: